openapi/interzoid.zipinfo/client.bal

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/url;
+import ballerina/lang.'string;
+
+# This is a generated connector for [Interzoid Zip Code Detailed Info API v1.0.0] (https://www.interzoid.com/services/getzipcodeinfo) OpenAPI specification.
+# The Interzoid Zip Code Detailed Info connector module provides detailed information for a given zip code, including city, state, latitude, longitude, area size, and various population demographics.
+# For additional help getting started with the API, visit [Interzoid Zip Code Detailed Info API] (https://www.interzoid.com/services/getzipcodeinfo).
+public isolated client class Client {
+    final http:Client clientEp;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials.
+    # Create an [Interzoid Account](https://www.interzoid.com/register) 
+    # and obtain tokens by log into [Interzoid Account](https://www.interzoid.com/account).
+    #
+    # + clientConfig - The configurations to be used when initializing the `connector`
+    # + serviceUrl - URL of the target service
+    # + return - An error at the failure of client initialization
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://api.interzoid.com") returns error? {
```

Comment:
The sample you have written in `Module.md` has APIKey. But `init` doesn't have such a parameter. Also, if this API doesn't have `securityScheme`, it doesn't mean it is unsecured. You can find a similar connector in https://github.com/ballerina-platform/ballerinax-openapi-connectors/blob/main/openapi/fraudlabspro.frauddetection/client.bal#L27-L38

Reply:
Yeah. We will remove the API keys from Module.md
/Thanks

Reply:
Added the API key as a required param in connector init.